resources/packs/gcp/gcp.lr

```diff
@@ -3,9 +3,6 @@ option go_package = "go.mondoo.com/cnquery/resources/packs/gcp"
 alias gcloud.organization = gcp.organization
 alias gcloud.project = gcp.project
 alias gcloud.resourcemanager.binding = gcp.resourcemanager.binding
-alias gcloud.compute = gcp.compute
```

Comment:
@chris-rock for some reason if those aliases stay in I get the following error when cnquery launches:
```
./cnquery run gcp -c "gcp.project.compute.networks{*}"😂                                                                                                                            
panic: failed to add resource info: could not merge resources because IDs don't match: , gcp.project

goroutine 1 [running]:
go.mondoo.com/cnquery/resources/packs/gcp/info.init.0()
        /Users/ivanmilchev/Projects/cnquery/resources/packs/gcp/info/info.go:27 +0xb8
```

If the aliases are deleted, then everything works correctly. I tried to figure out what the root cause of the error is but I am not entirely sure. Any suggestions what to do with this?